security/security-core/src/main/java/io/camunda/security/configuration/ConfiguredMapping.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.security.configuration;
+
+public class ConfiguredMapping {
```

Comment:
:wrench: Should this not do a sanity check, i.e. that claim name and value are both not empty?